trying to add new framework page

aneatrour opened the pull request:
new framework page

kgrons committed:
add Jekyll intro and resources

contribution from Andrea L. via feedback document

plampron2 committed:
fixes issue #96

GentrySteven committed:
Add files via upload

GentrySteven committed:
Merge pull request #1 from GentrySteven/GentrySteven-patch-1

Add files via upload

kateefly committed:
Merge pull request #112 from plampron2/issue#96

fixes issue #96

GentrySteven committed:
Add files via upload

Permalink in index removed.

Permalink for "About" page updated.

kateefly committed:
Merge pull request #113 from GentrySteven/issue#94

Fixes Issue#94 and Issue#95

kateefly commented:
@aneatrour This pull request only shows in your forked repository. Would you be able to open one in [https://github.com/DLFMetadataAssessment/Sandbox/pulls](https://github.com/DLFMetadataAssessment/Sandbox/pulls) instead?

kateefly committed:
Added permalink to Environmental Scan page. Also updated Ruby gems

dkulczak committed:
fixes issue #98

GentrySteven committed:
Add files via upload

Introductory page for the Tools sub-group. Will need to be reviewed.

kateefly committed:
Merge pull request #115 from kateefly/issue#101

Added permalink to Environmental Scan page. Also updated Ruby gems

kateefly committed:
Merge pull request #116 from GentrySteven/issue#114

Add files via upload

kgrons committed:
add skillbuilding front matter

thx to @kateefly for help!

kgrons committed:
add spaces under headers for table display

thanks for catching, @kateefly !

kateefly committed:
Merge pull request #89 from DLFMetadataAssessment/kgrons-patch-1

add Jekyll intro and resources

akhtars committed:
Migrate Benchmarks content in single page

akhtars committed:
Mark benchmarks survey as closed

kateefly committed:
Merge pull request #117 from DLFMetadataAssessment/issue#102

Migrate Benchmarks content in single page

kateefly committed:
Merge pull request #6 from DLFMetadataAssessment/master

Updating to make this in sync with master

kateefly committed:
Merge pull request #118 from aneatrour/master

Framework updates

kateefly committed:
Merge pull request #7 from DLFMetadataAssessment/master

More changes from master

kateefly committed:
Updated Framework page. Added permalink, bread crumbs and anchor tags, also fixed bullets

kateefly committed:
Updated Collaborations page

kateefly committed:
Forgot permalink for collaborations page

kateefly committed:
Updated links so they are relative, also updated gem file

kateefly committed:
Update collaborations with suggested edits

plampron2 committed:
Merge pull request #121 from DLFMetadataAssessment/issue#97

Updated Collaborations page

kateefly committed:
Updated Framework page to fix references to Metadata Support Group

akhtars committed:
Remove extra closing anchor tag

akhtars committed:
Merge pull request #120 from DLFMetadataAssessment/issue#104

Updated Framework page. Added permalink, bread crumbs and anchor tags…

GentrySteven committed:
Merge pull request #122 from DLFMetadataAssessment/issue#108

Updated links so they are relative, also updated gem file

kateefly committed:
Relative links in previous update were relative to main site and not Sandbox

kateefly committed:
Forgot to update the Tools page link on the main page

plampron2 committed:
Merge pull request #123 from DLFMetadataAssessment/issue#108

Relative links in previous update were relative to main site and not …

dkulczak committed:
fixes issue #98

dkulczak committed:
fixes issue #98

kateefly committed:
Merge pull request #128 from DLFMetadataAssessment/issue#124

Issue#124

dkulczak committed:
fixes issue #98

dkulczak committed:
fixes issue #98

dkulczak committed:
fixes issue #98

kateefly committed:
Removing name from contributors

kateefly committed:
Moved Metadata Analysis workshop to its own project page

kateefly committed:
Updated Publications section to include information on the 2019 Environmental Scan update

aneatrour committed:
Merge pull request #130 from DLFMetadataAssessment/issue#125

Moved Metadata Analysis workshop to its own project page

kateefly committed:
Merge pull request #129 from DLFMetadataAssessment/issue#98

fixes issue #98

GentrySteven committed:
Add files via upload

Fixes issue #126 and #127. 

Slight change to wording on the home page--text for contributors changed to "The people who have made this working group a success" (prior text implied all contributors contributed to the website only).

Noticed something when reviewing metadataworkshop.markdown--the fifth sentence reads "The updated curriculum included an introduction to the [Metadata Assessment Framework](/Framework) developed by the Metadata Working Group and will close by helping participants strategize ways to bring these tools back into their daily workflows and continue to train others they work with to do the same." 

Should the "will close" in this sentence  be changed to the past tense?

GentrySteven committed:
Add files via upload

GentrySteven committed:
Add files via upload

kateefly committed:
Attempting to fix merge issues

kateefly committed:
Merge branch 'GentrySteven-issue#127'

kgrons committed:
add general intro text

fix 2nd part of #99

kateefly committed:
Merge pull request #136 from DLFMetadataAssessment/kgrons-skillbuildingintrotext

add general intro text

kateefly committed:
Making sure layouts and page titles are consistent

kateefly committed:
Adjusted max width of the logo to 20 percent - this makes it look much better in mobile

kateefly committed:
Making sure titles and permalinks are capitalized in the same way

kateefly committed:
Merge pull request #138 from DLFMetadataAssessment/issue#135

Adjusted max width of the logo to 20 percent -

kateefly committed:
Merge pull request #131 from DLFMetadataAssessment/issue#101

Updated to include information on the 2019 Environmetal Scan update

kateefly committed:
Making sure collaborations page title and permalink are the same

kateefly committed:
Title and permalink for Environmental Scan page should be capitalized in the same way

plampron2 committed:
Merge pull request #137 from DLFMetadataAssessment/issue#133

Making sure layouts and page titles are consistent

plampron2 committed:
Merge pull request #139 from DLFMetadataAssessment/issue#101

Title and permalink for Environmental Scan page should be capitalized…

kateefly committed:
Added anchor tags to Skillbuilding page

akhtars committed:
Clarify that benchmarks survey is closed and being analyzed

kateefly committed:
Merge pull request #141 from DLFMetadataAssessment/issue#102

Clarify that benchmarks survey is closed and being analyzed

kateefly committed:
Merge pull request #140 from DLFMetadataAssessment/issue#99

Added anchor tags to Skillbuilding page

dependabot[bot] committed:
Bump rack from 1.6.11 to 1.6.12

Bumps [rack](https://github.com/rack/rack) from 1.6.11 to 1.6.12.
- [Release notes](https://github.com/rack/rack/releases)
- [Changelog](https://github.com/rack/rack/blob/master/CHANGELOG.md)
- [Commits](https://github.com/rack/rack/compare/1.6.11...1.6.12)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump nokogiri from 1.10.4 to 1.10.8

Bumps [nokogiri](https://github.com/sparklemotion/nokogiri) from 1.10.4 to 1.10.8.
- [Release notes](https://github.com/sparklemotion/nokogiri/releases)
- [Changelog](https://github.com/sparklemotion/nokogiri/blob/master/CHANGELOG.md)
- [Commits](https://github.com/sparklemotion/nokogiri/compare/v1.10.4...v1.10.8)

Signed-off-by: dependabot[bot] <support@github.com>

kateefly committed:
Merge pull request #144 from DLFMetadataAssessment/dependabot/bundler/nokogiri-1.10.8

Bump nokogiri from 1.10.4 to 1.10.8

kateefly committed:
Merge pull request #143 from DLFMetadataAssessment/dependabot/bundler/rack-1.6.12

Bump rack from 1.6.11 to 1.6.12

dependabot[bot] committed:
Bump rubyzip from 1.2.4 to 2.2.0

Bumps [rubyzip](https://github.com/rubyzip/rubyzip) from 1.2.4 to 2.2.0.
- [Release notes](https://github.com/rubyzip/rubyzip/releases)
- [Changelog](https://github.com/rubyzip/rubyzip/blob/master/Changelog.md)
- [Commits](https://github.com/rubyzip/rubyzip/compare/v1.2.4...v2.2.0)

Signed-off-by: dependabot[bot] <support@github.com>

kateefly committed:
Merge pull request #145 from DLFMetadataAssessment/dependabot/bundler/rubyzip-2.2.0

Bump rubyzip from 1.2.4 to 2.2.0

Commit:
Bring Sandbox in line with main site

xiaolima09 committed:
Create Resources

xiaolima09 committed:
Delete Resources

xiaolima09 committed:
add resources page

xiaolima09 committed:
Merge pull request #154 from xiaolima09/master

The resources page

xiaolima09 committed:
Update resources.markdown

kateefly committed:
Merge pull request #155 from xiaolima09/master

Update resources.markdown

xiaolima09 committed:
create Presentation and Publication Page

xiaolima09 committed:
Update publication.md

Update the content

xiaolima09 committed:
Update publication.md

xiaolima09 committed:
Set theme jekyll-theme-architect

xiaolima09 committed:
Update resources.markdown

xiaolima09 committed:
Update publication.md

xiaolima09 committed:
Update resources.markdown

xiaolima09 committed:
Add subpages underneath Projects

xiaolima09 committed:
Merge branch 'master' of https://github.com/xiaolima09/Sandbox

xiaolima09 committed:
Update project_archived.md

Update the content

xiaolima09 committed:
Update project_current.md

Update the content

xiaolima09 committed:
Update project_current.md

xiaolima09 committed:
Update project_archived.md

xiaolima09 committed:
Merge pull request #156 from xiaolima09/master

Update Sandbox pages

GentrySteven committed:
Fixes issues 147, 149

kateefly committed:
Merge pull request #158 from GentrySteven/issue#149

Fixes issues 147

kateefly committed:
Merge branch 'master' into issue#146

kateefly committed:
Update _config.yml

kateefly committed:
Merge pull request #153 from DLFMetadataAssessment/issue#146

Bring Sandbox in line with main site

Commit:
Updating header and making sure entries are in the same folder

kateefly committed:
Merge pull request #159 from DLFMetadataAssessment/issue#157

Updating header and making sure entries are in the same folder

GentrySteven committed:
Update archived-environmental-scan.md

Just edited the markdown file directly, since it's such a small change.

Fixes issue 160

kateefly committed:
Merge pull request #161 from DLFMetadataAssessment/Issue#160

Update archived-environmental-scan.md

Commit:
Fixed header to point to Sandbox instead of main site

kateefly committed:
Merge pull request #163 from DLFMetadataAssessment/issue#162

Fixed header to point to Sandbox instead of main site

plampron2 committed:
fixes issue#172

plampron2 committed:
fixes issue#172 (#1)

GentrySteven committed:
Delete dlf-aig.png

GentrySteven committed:
Update Sandbox Image

Added SANDBOX text to header image, to differentiate sandbox from production site.

kateefly committed:
Merge pull request #173 from plampron2/issue#172

fixes issue#172

kateefly committed:
Merge pull request #174 from GentrySteven/issue#164

Issue#164

kateefly committed:
Merge pull request #8 from DLFMetadataAssessment/master

Bring up to date with origin/master

Commit:
Update projects pages with links to archived and current. Move info about projects to appropriate pages

plampron2 committed:
Merge branch 'master' of https://github.com/DLFMetadataAssessment/Sandbox

plampron2 committed:
fixes broken map clearinghouse link issue#169

plampron2 committed:
fixes broken map cleainghouse link issue#169

kateefly committed:
Merge pull request #176 from plampron2/issue#169

Issue#169

plampron2 committed:
fixes issue#170 - remove collaborations from header, remove page from entries, remove collaboration link from index.md

mbolam committed:
fixes content and structure from issues #168

Commit:
Updated publication entry with info from Mike

kateefly committed:
Merge pull request #178 from DLFMetadataAssessment/issue#168

fixes content and structure from issues #168

gabigulya committed:
Merge pull request #175 from kateefly/issue#167

Issue#167

GentrySteven committed:
Add files via upload

GentrySteven committed:
Fixes Issue #166

GentrySteven committed:
Delete take-part.markdown

GentrySteven committed:
Fixes issue #166

kateefly committed:
Merge pull request #182 from GentrySteven/issue#179

Fixes Issue #179

kateefly committed:
Merge pull request #184 from GentrySteven/issue#166

Issue#166

kateefly committed:
Merge pull request #177 from plampron2/issue#170

Issue#170

dependabot[bot] committed:
Bump nokogiri from 1.11.3 to 1.11.4

Bumps [nokogiri](https://github.com/sparklemotion/nokogiri) from 1.11.3 to 1.11.4.
- [Release notes](https://github.com/sparklemotion/nokogiri/releases)
- [Changelog](https://github.com/sparklemotion/nokogiri/blob/main/CHANGELOG.md)
- [Commits](https://github.com/sparklemotion/nokogiri/compare/v1.11.3...v1.11.4)

Signed-off-by: dependabot[bot] <support@github.com>

GentrySteven committed:
fixes issue#0191

GentrySteven committed:
fixes issue #195

kateefly committed:
Merge pull request #204 from GentrySteven/issue#0195

Issue#0195

kateefly committed:
Merge pull request #185 from DLFMetadataAssessment/dependabot/bundler/nokogiri-1.11.4

Bump nokogiri from 1.11.3 to 1.11.4
